torch/_functorch/aot_autograd.py

```diff
@@ -405,6 +405,13 @@ class OutputAliasInfo:
     #   here, this refers to the index of the *direct* traced
     base_idx: Optional[int]
 
+    def __eq__(self, other):
+        if not isinstance(other, OutputAliasInfo):
+            return NotImplemented
+        return (self.output_type == other.output_type and
+                self.base_idx == other.base_idx and
+                (issubclass(self.raw_type, other.raw_type) or issubclass(other.raw_type, self.raw_type)))
```

Comment:
This is surprising

Reply:
Yeah... I'll try to fix this for real (should have at least left a comment here). What I should be doing is grabbing the raw_type off the the original input to get the synthetics base's raw type. I spent 10 minutes trying to do that and it was painful, so I ended up doing this instead. I'm trying to remember exactly where my metadata check failed, but I think one had a type of `FakeTensor` and the other was `torch.Tensor`.

Reply:
yeaa i was wrong, not necessary